src/parse.rs

```diff
@@ -277,6 +278,11 @@ impl<'a> FirstPass<'a> {
         let begin_list_item = false;
         let last_line_blank = false;
         let allocs = Allocations::new();
+        let lookup_table = if options == Options::empty() {
```

Comment:
Only the table and strikethrough options add new tokens for now, so we could just check those, right?

Reply:
Yeah I’ll change that